src/Store/Actions/Actions.js

```diff
@@ -183,7 +183,7 @@ export const clearInventoryStore = () =>({
     payload: []
 });
 
-export const clearVulnerabilitiesStore = () =>({
-    type: ActionTypes.CLEAR_VULNERABILITY_STORE,
+export const clearCVEsStore = () =>({
+    type: ActionTypes.CLEAR_CVES_STORE,
```

Comment:
`CLEAR_CVES_STORE` should be changed to `CLEAR_CVEs_STORE`

Reply:
in the `ActionTypes` is defined as `CLEAR_CVES_STORE` cause the constant should be camel case